src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -24,111 +25,195 @@ def setUp(self):
+            oclass = self.ior_cmd.dfs_oclass.value
 
+        # Exclude ranks [0, 3, 4]
+        rank = [0, 3, 4]
         for val in range(0, num_pool):
             pool[val] = TestPool(self.context,
                                  dmg_command=self.get_dmg_command())
             pool[val].get_params(self)
-            # Split total SCM and NVME size for creating multiple pools.
-            pool[val].scm_size.value = int(pool[val].scm_size.value /
-                                           num_pool)
-            pool[val].nvme_size.value = int(pool[val].nvme_size.value /
-                                            num_pool)
             pool[val].create()
-            pool_uuid.append(pool[val].uuid)
             self.pool = pool[val]
-            if data:
-                self.run_ior_thread("Write", self.ior_dfs_oclass[0],
-                                    self.ior_apis[0], self.ior_test_sequence[0])
-
-        # Exclude and reintegrate the pool_uuid, rank and targets
-        for val in range(0, num_pool):
-            self.pool = pool[val]
-            self.pool.display_pool_daos_space("Pool space: Beginning")
-            pver_begin = self.get_pool_version()
-            self.log.info("Pool Version at the beginning %s", pver_begin)
-            if server_boot is False:
-                output = self.dmg_command.pool_exclude(self.pool.uuid,
-                                                       rank)
+            if reint_during_aggregation is True:
+                test_seq = self.ior_test_sequence[1]
+                self.pool.set_property("reclaim", "disabled")
             else:
-                output = self.dmg_command.system_stop(ranks=rank)
-                self.pool.wait_for_rebuild(True)
+                test_seq = self.ior_test_sequence[0]
+            if data:
+                self.run_ior_thread("Write", oclass, test_seq)
+                self.run_mdtest_thread()
+
+        # Exclude all the ranks
+        random_pool = random.randint(0, (num_pool-1))
+        for _ in range(0, self.loop_test_cnt):
+            for val, _ in enumerate(rank):
+                self.pool = pool[random_pool]
+                self.pool.display_pool_daos_space("Pool space: Beginning")
+                pver_begin = self.get_pool_version()
+                self.log.info("Pool Version at the beginning %s", pver_begin)
+                if server_boot is False:
+                    if (reint_during_rebuild is True and val == 0):
+                        # Exclude rank 5
+                        output = self.dmg_command.pool_exclude(self.pool.uuid,
+                                                               "5")
+                        self.log.info(output)
+                        self.is_rebuild_done(3)
+                        self.assert_on_rebuild_failure()
+                    if reint_during_aggregation is True:
+                        self.pool.set_property("reclaim", "time")
+                        time.sleep(90)
```

Comment:
Same as above.

Reply:
Will remove it... Same comment as above.

Reply:
Done.